### PR TITLE
fix(DTFS2-7694): facility issuance reconciliation with TFM

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction :pencil2:
When a facility is issued payload response received from ACBS is not being reconciliated back to `tfm-facilities` acbs object.

## Resolution :heavy_check_mark:
* Updated to `facilityIdentifier` as similar variables were updated in `abcs` microservice.

## Miscellaneous ➕ 
* Added unit test cases.
* Lint fixes.
* Documentation improvement.

### CODE REVIEW

File: trade-finance-manager-api/src/v1/controllers/acbs.controller.js [hunk 1]
```diff
@@ -140,8 +140,8 @@ const updateDealAcbs = async (taskOutput) => {
  * @param {Object} FFR Facility fixed fee record
  * @returns {Object} ACBS returned response
  */
-const updateIssuedFacilityAcbs = ({ facilityId, issuedFacilityMaster, facilityLoan, facilityFee }) =>
-  tfmController.updateFacilityAcbs(facilityId, {
+const updateIssuedFacilityAcbs = ({ facilityIdentifier, issuedFacilityMaster, facilityLoan, facilityFee }) =>
```
Comment (Zainzzkk) [inline trade-finance-manager-api/src/v1/controllers/acbs.controller.js:143]: do we need to update a test?

Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/controllers/acbs.controller.js]: Have added.

File: trade-finance-manager-api/src/v1/controllers/acbs.controller.test.js [hunk 2]
```diff
@@ -39,8 +49,117 @@ const mockACBSTaskLink = {
   purgeHistoryDeleteUri: 'acbs',
 };
 
+const mockAcbsFacilityIssueTask = {
```
Comment (Zainzzkk) [inline trade-finance-manager-api/src/v1/controllers/acbs.controller.test.js:52]: could this be a test mock?

Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/controllers/acbs.controller.test.js]: Added.

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**